src/InstallationValidator.Core/Presentation/SimulationComparisonPresenter.cs

```diff
@@ -24,7 +24,7 @@ public interface ISimulationComparisonPresenter : IDisposablePresenter, ILoggerP
    public class SimulationComparisonPresenter : AbstractDisposablePresenter<ISimulationComparisonView, ISimulationComparisonPresenter>, ISimulationComparisonPresenter
    {
       private readonly FolderDTO _firstFolderDTO = new FolderDTO();
-      private readonly FolderDTO _secondFolderDTO = new FolderDTO();
+      private readonly FolderDTO _secondFolderDTO = new FolderDTO(limitFolderLength:true);
```

Comment:
To apply that to only the folder where the report is created means that the presenter knows that this dto is the one who will get the report.

Alternative is that we just limit both folders to 130 characters.